lib/linter-views.coffee

```diff
@@ -4,49 +4,44 @@ Message = require './views/message'
 
 class LinterViews
   constructor: (@linter) ->
-    @showPanel = true # Altered by config observer in linter-plus
-    @showBubble = true # Altered by the config observer in linter-plus
-    @underlineIssues = true # Altered by config observer in linter-plus
+    @showPanel = true
+    @showBubble = true
+    @underlineIssues = true
 
     @messages = new Set
+    @lineMessages = new Set
     @markers = []
     @statusTiles = []
 
-    @tabs = new Map
-    @tabs.set 'line', new BottomTab()
-    @tabs.set 'file', new BottomTab()
-    @tabs.set 'project', new BottomTab()
+    @tabs = {}
+    @tabs['Line'] = new BottomTab()
+    @tabs['File'] = new BottomTab()
+    @tabs['Project'] = new BottomTab()
 
     @panel = document.createElement 'div'
     @bubble = null
     @bottomStatus = new BottomStatus()
 
-    @tabs.get('line').initialize 'Line', => @changeTab('line')
-    @tabs.get('file').initialize 'File', => @changeTab('file')
-    @tabs.get('project').initialize 'Project', => @changeTab('project')
+    @tabs['Line'].initialize 'Line', => @changeTab('Line')
+    @tabs['File'].initialize 'File', => @changeTab('File')
+    @tabs['Project'].initialize 'Project', => @changeTab('Project')
```

Comment:
You could work around that hacking function scoping rules... And that's what `do` keyword is for.
```coffeescript
for k,v of @tabs
    do (k, v) =>
        v.initialize k, => @changeTab(k)
```